jest.config.ts

```diff
@@ -0,0 +1,9 @@
+import type { Config } from '@jest/types';
+// Sync object
+const config: Config.InitialOptions = {
+  verbose: true,
+  transform: {
+    '^.+\\.(ts)?$': 'ts-jest',
+  },
+};
+export default config;
```

Comment:
Add a comment to your teammates with a brief explanation of these things that may need explanation, or you can add it to the issue description.

Reply:
Repaired.